### PR TITLE
feat(wallet): add getInternal addresses in jsonrpc

### LINKED ISSUE #1634: [Wallet] Add JsonRPC method to retrieve internal addresses
Currently, the `get_addresses` method only returns only the external addresses.

A possible implementation might be to include an optional field called `external` to that method.

### PR DESCRIPTION
**Pull request by girazoki**
Closes #1634

### CODE REVIEW

File: wallet/src/actors/worker/handlers/get_addresses.rs [hunk 1]
```diff
@@ -9,6 +9,8 @@ pub struct GetAddresses(
     pub u32,
     /// Limit
     pub u32,
+    /// External
+    pub bool,
```
Comment (tmpolaczyk) [inline wallet/src/actors/worker/handlers/get_addresses.rs:13]: Refactor this into a struct with name fields please

```suggestion
    /// External
    pub external: bool,
```

Comment (mariocao) [reply wallet/src/actors/worker/handlers/get_addresses.rs:13]: Yep, that's a known a easy refactor! :)

File: wallet/src/repository/wallet/mod.rs [hunk 2]
```diff
@@ -373,7 +373,34 @@ where
...
+        for index in range.rev() {
+            let address = self.get_address(account, keychain, index)?;
+            addresses.push((*address).clone());
+        }
+
+        Ok(model::Addresses { addresses, total })
+    }
+
+    /// Return a list of the generated internal addresses that.
+    pub fn internal_addresses(&self, offset: u32, limit: u32) -> Result<model::Addresses> {
```
Comment (mariocao) [inline wallet/src/repository/wallet/mod.rs:391]: I think this function can be refactored together with `external_addresses`.

Maybe creating a generic non-public `addresses` function with the `limit`, `offset` and `keychain` as argument.

**Review by mariocao — CHANGES_REQUESTED** [review]

File: wallet/src/actors/worker/handlers/get_addresses.rs (hunk 1 above)
Comment (girazoki) [reply wallet/src/actors/worker/handlers/get_addresses.rs:13]: this is a common issue with all handlers. I would change them all in the same issue, not on this one obviously

**Review by tmpolaczyk — APPROVED** [review]

**Review by mariocao — APPROVED** [review]

**Pull request merged**